### PR TITLE
[DG] Update Design section (Model still in progress)

### PR DESCRIPTION
**Pull request by lyeyixian**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -62,11 +62,11 @@ The sections below give more details of each component.
 ![Structure of the UI Component](images/UiClassDiagram.png)
 
 **API** :
-[`Ui.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/Ui.java)
+[`Ui.java`](https://github.com/AY2021S1-CS2103-T14-3/tp/blob/master/src/main/java/seedu/address/ui/Ui.java)
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
+The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `TravelPlannerPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
```
Comment (jiaweiteo) [inline docs/DeveloperGuide.md:67]: Need add TravelPlanPanel, TravelPlanObjectListPanel?

File: docs/diagrams/DeleteSequenceDiagram.puml [hunk 2]
```diff
@@ -3,66 +3,66 @@
...
 
-LogicManager -> AddressBookParser : parseCommand("delete 1")
-activate AddressBookParser
+LogicManager -> WanderlustParser : parseCommand("delete -activity 1")
+activate WanderlustParser
 
 create DeleteCommandParser
-AddressBookParser -> DeleteCommandParser
+WanderlustParser -> DeleteCommandParser
 activate DeleteCommandParser
```
Comment (jiaweiteo) [inline docs/diagrams/DeleteSequenceDiagram.puml:24]: DeleteCommandParser is Called WanderlustDeleteCommandParser for all

**Review by jiaweiteo — APPROVED** [review]
LGTM, just need to change deleteCommandParser to wanderlustDeleteCommandParser in the delete sequence diagram.

File: docs/diagrams/DeleteSequenceDiagram.puml (hunk 2 above)
Comment (lyeyixian) [reply docs/diagrams/DeleteSequenceDiagram.puml:24]: Actually I changed all to without all the Wanderlust in the front. Tsk didn't read my telegram message. YOU SHOULD CHANGED!!! NOT ME!!! HAHAHAHA

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (lyeyixian) [reply docs/DeveloperGuide.md:67]: They are included in the etc XD or else too long lah, or you think I should change the commandbox all that to travelplan thing?

File: docs/diagrams/DeleteSequenceDiagram.puml (hunk 2 above)
Comment (jiaweiteo) [reply docs/diagrams/DeleteSequenceDiagram.puml:24]: OK MY BAD

**Pull request merged**